esbuild/assets.config.js

```diff
@@ -0,0 +1,49 @@
+const { copy } = require('esbuild-plugin-copy')
+const { sassPlugin } = require('esbuild-sass-plugin')
+const { clean } = require('esbuild-plugin-clean')
+const esbuild = require('esbuild')
+const path = require('path')
+const { glob } = require('glob')
+
+const buildAdditionalAssets = buildConfig =>
+  esbuild.build({
+    outdir: buildConfig.assets.outDir,
+    plugins: [
+      copy({
+        resolveFrom: 'cwd',
+        assets: buildConfig.assets.copy,
+      }),
+    ],
+  })
+
+const buildAssets = buildConfig => {
+  return esbuild.build({
+    entryPoints: buildConfig.assets.entryPoints,
+    outdir: buildConfig.assets.outDir,
+    entryNames: '[ext]/app',
+    minify: buildConfig.isProduction,
+    sourcemap: !buildConfig.isProduction,
+    platform: 'browser',
+    target: 'es2015',
```

Comment:
tsconfig uses 2018 for node and a cursory look at caniuse.com makes me wonder if we can use 2018 here too? what do people think?

Reply:
I'd agree with a later version, even GDS has basically dropped IE support for example.